src/BTR3.ohm

```diff
@@ -74,7 +74,7 @@ BTR3 {
 
     amountOpt = optSign digit*
     itemCount = digit*
-    fundsType = alnum?
+    fundsType = ("0".."2" | "S" | "V" | "D" | "Z") ?
```

Comment:
Again, hardening the definition, there is a specific set of values for this [pg. 29]